ledger/participant-integration-api/src/main/scala/platform/store/dao/events/ContractsTable.scala

```diff
@@ -23,6 +23,12 @@ private[events] abstract class ContractsTable extends PostCommitValidationData {
   private def deleteContract(contractId: ContractId): Vector[NamedParameter] =
     Vector[NamedParameter]("contract_id" -> contractId)
 
+  private val nullifyPastKeysQuery =
```

Comment:
I think this could benefit from some comments somewhere on why this is a safe thing to do.

Reply:
Done